Google.Cloud.EntityFrameworkCore.Spanner/Query/Internal/SpannerQuerySqlGenerator.cs

```diff
@@ -122,5 +137,54 @@ protected override Expression VisitSqlFunction(SqlFunctionExpression sqlFunction
 
             return base.VisitSqlFunction(sqlFunctionExpression);
         }
+
+        protected virtual Expression VisitContains(SpannerContainsExpression containsExpression)
+        {
+            var valueRequiresParentheses = ValueRequiresParentheses(containsExpression.Values);
+            Visit(containsExpression.Item);
+            Sql.Append(containsExpression.IsNegated ? " NOT IN " : " IN ");
+
+            if (valueRequiresParentheses)
+            {
+                Sql.Append("(");
+            }
+
+            if (containsExpression.Values is SqlConstantExpression constantValuesExpression
+                && constantValuesExpression.Value is IEnumerable constantValues)
+            {
+                var first = true;
+                foreach (var item in constantValues)
+                {
+                    if (!first)
+                    {
+                        Sql.Append(", ");
+                    }
+
+                    first = false;
+                    Sql.Append(constantValuesExpression.TypeMapping?.GenerateSqlLiteral(item) ??
+                               item?.ToString() ?? "NULL");
+                }
```

Comment:
Could we use `String.Join(..)` for this? So something like `String.Join(", ", constantValues.Select(item => constantValuesExpression.TypeMapping?.GenerateSqlLiteral(item) ??
                               item?.ToString() ?? "NULL")`

Reply:
@olavloite The `Sql` variable is not actually a string, it's an `IRelationalCommandBuilder`.

Reply:
Ack